internal/store/migrations.go

```diff
@@ -45,6 +45,90 @@ var migrations = []migration{
 			return err
 		}
 
+		return nil
+	}},
+	{semver.MustParse("0.1.0"), semver.MustParse("0.2.0"), func(e execer) error {
```

Comment:
Strictly speaking, I could force you all to just re-create your databases while we're <1.0.0, but I figure I'd exercise the migration system for now.

Reply:
Yeah, I am totally fine with this.